### DISCUSSION TITLE
Concourse Jira release resource

### DISCUSSION THREAD

**Discussion by richardforjoejnr** [concourse/concourse#8797] [Ideas]
Hi, i'm looking for some help/direction. I would like to create a concourse resource which uses Jira release feature to trigger a pipeline. So a resource similar to github release resource: https://github.com/concourse/github-release-resource

I'm not sure where to start with this so some guidance would be much appreciated.  First has this been done already as i couldn't find any resource on this. Secondly where should i start. Is there an example repo for a jira resource i could use?

Jira release is basically based on a Jira webhook being fired when a release is published in Jira, so this resource needs to be able to listen to that and get relevant info from the webhook eg Release version, dates, description. 

Jira webhooks: https://developer.atlassian.com/server/jira/platform/webhooks/

**Comment by rpgeddam** [comment]
Saw this in a different discussion thread: https://concoursetools.readthedocs.io/en/stable/quickstart.html
Perhaps it'd be useful for you to get started?

**Reply by DanielJonesEB** [reply]
TIL the UK's intelligence agency uses Concourse!

**Reply by richardforjoejnr** [reply]
Nice one. Trying it out now. Thank you

**Comment by DanielJonesEB** [comment]
You might also like to look at [Smuggler](https://github.com/redfactorlabs/concourse-smuggler-resource) which basically takes its code at runtime through the pipeline definition, so you can iterate more quickly whilst trying stuff out. Not recommended for production!

**Reply by richardforjoejnr** [reply]
Super. Thank you, will check this out
